src/analysis/convex/extreme.lean

```diff
@@ -0,0 +1,286 @@
+/-
+Copyright (c) 2021 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import linear_algebra.affine_space.independent
+import algebra.big_operators.basic
+import analysis.convex.topology
+
+/-!
+# Extreme sets
+This file defines extreme sets and extreme points for set in a real normed space.
+## References
+See chapter 8 of [Convexity][simon2011]
+TODO:
+- add exposed sets to this file.
+- define convex independence and prove lemmas related to extreme points.
+-/
```

Comment:
The documentation style guide seems to suggest that you should have line breaks here
```suggestion
/-!
# Extreme sets

This file defines extreme sets and extreme points for set in a real normed space.

## References

See chapter 8 of [Convexity][simon2011]

TODO:
- add exposed sets to this file.
- define convex independence and prove lemmas related to extreme points.
-/
```